static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.spec.js

```diff
@@ -59,7 +59,7 @@ describe('initialise', () => {
             _disableAjaxTimeout: false,
             _mediaTypePriceGranularity: {},
             _priceGranularity: 'custom',
-            _publisherDomain: 'http://localhost',
+            _publisherDomain: 'http://test-url.com',
```

Comment:
See comment above. Seems like this change was required because of the changes to `package.json` but it's unclear why that change is necessary.